core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -274,6 +279,31 @@ var TopicEditorPage = function() {
       'Create subtopic modal taking too long to disappear.');
   };
 
+  this.addConceptCardToSubtopicExplanation = async function(skillName) {
+    var pageEditorInput = element(by.css('.protractor-test-edit-html-content'));
+    await action.click('RTE input', pageEditorInput);
+    var conceptCardButton = element(
+      by.cssContainingText('.cke_button', 'Insert Concept Card Link'));
```

Comment:
Can you use a `protractor-test-*` HTML class to identify this element? If you can't add a class to this element, see if you can add a class to a parent element so you can do `element(by.css('.protractor-test-*')).element(...)`

Reply:
Done